release-team/role-handbooks/release-signal/README.md

```diff
@@ -0,0 +1,562 @@
+      * [Once the Code Freeze starts - one week later](#once-the-code-freeze-starts---one-week-later)
+      * [Week 1 of Code Freeze until Code Thaw starts](#week-1-of-code-freeze-until-code-thaw-starts)
+    * [Code Thaw](#code-thaw)
+  * [Release Cutting - Go or No-Go](#release-cutting---go-or-no-go)
+  * [Judging State of Testgrid dashboards](#judging-state-of-testgrid-dashboards)
+  * [Blocking vs. Informing Dashboards](#blocking-vs-informing-dashboards)
+  * [Opening Issues](#opening-issues)
+    * [Decision Tree](#decision-tree)
+    * [Closing Issues](#closing-issues)
+  * [Special high risk test categories to monitor](#special-high-risk-test-categories-to-monitor)
+    * [Scalability tests](#scalability-tests)
+  * [Reporting](#reporting)
+  * [Tips and Tricks of the game](#tips-and-tricks-of-the-game)
+    * [A Tour of CI on the Kubernetes Project](#a-tour-of-ci-on-the-kubernetes-project)
+    * [Checking test dashboards](#checking-test-dashboards)
+    * [Priority Labels](#priority-labels)
+    * [Milestones](#milestones)
+<!-- TOC -->
+
+## Overview
+
+The release signal team is responsible for making sure that **both** Issues (`is:issue`) and Pull Requests (PRs) (`is:pr`), which
+are targeted for the ongoing release cycle, are dealt with in a timely fashion. It also assumes the responsibility of the quality
+gate for the release, checking the health status of testgrid dashboards. This team is a combination of old CI Signal and Bug
+Triage sub-team. Further responsibilities are:
+
+- Continuously monitoring various e2e tests in sig-release
+  dashboards ([master-blocking](https://testgrid.k8s.io/sig-release-master-blocking), [master-informing](https://testgrid.k8s.io/sig-release-master-informing), `release-x.y-blocking/informing` (
+  x.y being the current release)) throughout the release cycle
+- Providing early and ongoing signals on release and test health to both the Release team and various SIGs
+- Ensuring that all release-blocking tests provide a clear Go/No-Go signal for the release
+- Flagging regressions as close to the source as possible, i.e., as soon as the offending code was merged
+- Filing issues proactively for test failures and flakes, triaging issues to appropriate SIG/owner, following up on status and
+  verifying fixes on newer test runs
+    - See also: [How to escalate issues](#how-to-escalate)
+- Studying patterns/history of test health from previous releases and working closely with SIGs and test owners to
+    - Understand and fix the test for the current release
+    - Understand if the test needs to be on the release blocking dashboard
+    - Work with SIG-Testing on any possible test infra improvement to help improve the test pass rate
+- Making recommendations to SIG-Release for promoting and demoting release blocking and merge blocking tests as per
+  the [Blocking Jobs Criteria](/release-blocking-jobs.md)
+- Listing open issues and PRs targeted for the ongoing release cycle
```

Comment:
reporting issues and PRs right? we already mentioned this

Reply:
Done, slimmed the list.